src/network/clientpackethandler.cpp

```diff
@@ -1013,12 +1016,19 @@ void Client::handleCommand_AddParticleSpawner(NetworkPacket* pkt)
 
 void Client::handleCommand_DeleteParticleSpawner(NetworkPacket* pkt)
 {
-	u32 id;
-	*pkt >> id;
+	u32 server_id;
+	*pkt >> server_id;
+
+	u32 client_id;
+	auto i = m_particles_server_to_client.find(server_id);
+	if (i != m_particles_server_to_client.end())
+		client_id = i->second;
+	else
+		return;
```

Comment:
show a warning when this happens?

Reply:
Better not. What if the particle spawner wasn't sent to the client on join? (already active before the client joined)

Reply:
The current code, just ignores any unknown particle spawner IDs, so it seems reasonable to just keep it that way.